addons/web/tooling/_eslintrc.json

```diff
@@ -46,6 +51,18 @@
     "py": "readonly",
     "FullCalendar": "readonly",
     "ClipboardJS": "readonly",
-    "globalThis": "readonly"
+    "globalThis": "readonly",
+    "Modal": "readonly",
+    "Dropdown": "readonly",
+    "ScrollSpy": "readonly",
+    "Tooltip": "readonly",
+    "Collapse": "readonly",
+    "Alert": "readonly",
+    "module": "readonly",
+    "chai": "readonly",
+    "describe": "readonly",
+    "it": "readonly",
+    "mocha": "readonly",
+    "DOMPurify": "readonly"
```

Comment:
These seem to be used in very specific places and they should probably be defined as globals only in those files (especially because the files that declare them are not available in most places because they're only in very specific assets bundles)